### DISCUSSION TITLE
Title and Tool bar gone

### DISCUSSION THREAD

**Discussion by ekoGameDev** [atom/atom#23636] [Q&A] [answered]
![Capture](https://user-images.githubusercontent.com/100079876/154847601-d8ee0dc7-a839-4176-8ab0-1461c4f4934a.PNG)
I do not know how but I managed to make not only my tool bar but my title bar disappear. I have not found a fix to make them come back. I have not installed any packages to Atom or changed any configurations, I do not know how I managed to do this.

**Comment by icecream17** [comment, marked as answer]
maybe these could help
https://github.com/atom/atom/discussions/23467
https://stackoverflow.com/questions/29159379/atom-menu-is-missing-how-do-i-re-enable
